src/shared/components/query/QueryStore.ts

```diff
@@ -1273,6 +1273,7 @@ export class QueryStore
 		if ((_window as any).serverVars) {
 			// Populate OQL
 			this.geneQuery = normalizeQuery((_window as any).serverVars.theQuery);
+			this.genesetQuery = normalizeQuery((_window as any).serverVars.genesetIds);

```

Comment:
Just confirming - `normalizeQuery` was written for OQL. it works for this too?

Reply:
Yes!